testing/platform/temporaltesting/temporal.go

```diff
@@ -0,0 +1,49 @@
+package temporaltesting
+
+import (
+	"context"
+	"io"
+
+	"github.com/stretchr/testify/require"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/testsuite"
+)
+
+var defaultNamespace = "temporaltesting-default-namespace"
+
+type TemporalT interface {
+	require.TestingT
+	Cleanup(func())
+}
+
+type TemporalServer struct {
+	*testsuite.DevServer `json:"-"`
+	defaultNamespace     string
+}
+
+func CreateTemporalServer(t TemporalT, w io.Writer) *TemporalServer {
+	srv, err := testsuite.StartDevServer(context.Background(), testsuite.DevServerOptions{
+		ClientOptions: &client.Options{Namespace: defaultNamespace},
+		Stdout:        w,
+		Stderr:        w,
+	})
+	if err != nil {
+		require.Failf(t, "failed to start temporal dev server: %s", err.Error())
+	}
+	t.Cleanup(func() {
+		_ = srv.Stop()
+	})
+	return &TemporalServer{DevServer: srv, defaultNamespace: defaultNamespace}
+}
+
+func (s *TemporalServer) Client(ctx context.Context) client.Client {
+	return s.Client(ctx)
+}
```

Comment:
Can we have more than one client ? for // testing

Reply:
testsuite.DevServer pre-emptively creates a client configured to communicate with the DevServer and this method gives access to that particular preconfigured client. But there's nothing stopping the tester from using the `DefaultNamespace` and `HostPort` methods to configure additonal clients using any of the constructors provided by the go-sdk client package.